README.md

```diff
@@ -111,6 +112,7 @@ jobs:
           update-comment-if-one-exists: true            # Default: true
```

Comment:
I can't make a comment in the right place because it didn't change, but the `uses: im-open/process-dotnet-test-results@v2.0.2` for this step needs to be updated with the new version number.  There is a section below this for specifying additional behavior that also needs to have the version updated.